### DISCUSSION TITLE
how to remove maintenance mode

### DISCUSSION THREAD

**Discussion by arodriguez401** [JulianPrieber/littlelink-custom#443] [Issues]
After enabling maintenance mode, I find myself unable to log back in. I'm seeking assistance on how to disable this feature.

Platform: `Ubuntu 22.04 LTS`
Server type: `NGINX/Apache2 ...`
PHP version: `8.1`

[//]: # (Please make sure you have read the system requirements at https://llc.ovh/req before submitting your issue.)

**Comment by lastsamurai26** [comment]
Hey, 

you can only disable the Maintenance Mode via FTP or SSH editing the .env file 
search for MAINTENANCE_MODE= -> set this to false

**Comment by arodriguez401** [comment]
![image](https://github.com/LinkStackOrg/LinkStack/assets/119558965/b1855cdf-bd45-4daf-9dc9-0a7590bfe0bc)

Maintenance mode is set to false

**Reply by arodriguez401** [reply]
![image](https://github.com/LinkStackOrg/LinkStack/assets/119558965/4bbbcd9e-4c8e-4361-aaa6-336c45f0043b)
It still shows the maintenance page after i enabled it in the admin panel

**Reply by JulianPrieber** [reply]
You might have to delete the maintenance file in the storage folder. 

If you can access the admin panel you have to untick maintenance mode in the config, that does the same.

**Reply by arodriguez401** [reply]
I wasn't able to getting into the admin panel and untick the maintenance mode in the config. I just re-ran the docker-compose and had to recreate everything.
